CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ### Features
 
+* `[jest-config]` According to ([#5828](https://github.com/facebook/jest/issues/5828)), moves `deprecationEntries` from
+  `jest-config` to `jest-validate`.
+  ([#6067](https://github.com/facebook/jest/pull/6067))
+* `[jest-validate]` According to ([#5828](https://github.com/facebook/jest/issues/5828)), moves `deprecationEntries` from
```

Comment:
also, I don't think we need to link to #5828 at all, linking to this PR is enough (then peaople can follow the trail if they are curious)